src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -13,9 +16,19 @@
 	public SelectTestFigureOptionListener(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
-
 	@Override
 	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		if(e.getActionCommand().equals("Figure Joe 1")){
+			FiguresJoe.figureScript1(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("Figure Joe 2")){
+			FiguresJoe.figureScript2(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("Rectangle")){
+			DrawFactory.drawRectangle (-150,-150,150,200).execute(driverManager.getCurrentDriver());
+		}
+		else if(e.getActionCommand().equals("Triangle")){
+			DrawFactory.drawTriangle (-150,-150,150).execute(driverManager.getCurrentDriver());
+		}
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.